polaris-service/src/main/java/org/apache/polaris/service/config/PolarisApplicationConfig.java

```diff
@@ -170,7 +169,12 @@ public TaskHandlerConfiguration getTaskHandler() {
 
   @JsonProperty("featureConfiguration")
   public void setFeatureConfiguration(Map<String, Object> featureConfiguration) {
-    this.configurationStore = new DefaultConfigurationStore(featureConfiguration);
+    this.featureConfiguration = featureConfiguration;
+  }
+
+  @JsonProperty("realmFeatureConfiguration")
```

Comment:
What does the config file look like in this case? It seems to be something like...

```
featureConfiguration:
  SOME_FEATURE: true

realmFeatureConfiguration:
  realm_1:
    SOME_FEATURE: true
  realm_2:
    SOME_FEATURE: false
```

Can we do any validation that these realms actually exist?

Reply:
yeah, that's the proposed configuration structure in the yaml file. No, we can't validate that the realms exist currently - there is no agreed upon interface for storing/retrieving realms and the only persistence API doesn't know anything about realms.